InstallSpinnaker.sh

```diff
@@ -612,6 +623,10 @@ if [ "x$GOOGLE_PROJECT_ID" != "x" ]; then
   write_default_value "SPINNAKER_GOOGLE_PROJECT_ID" $GOOGLE_PROJECT_ID
 fi
 
+if [ "x$GOOGLE_CREDS_PATH" != "x" ]; then
```

Comment:
Removed flags and google initialization, let's just point them to /etc/default/spinnaker and have them put in the defaults. I will update my doc to reflect that.